webui/react/src/e2e/models/pages/SignIn.ts

```diff
@@ -8,7 +9,7 @@ import { PageComponent } from 'e2e/models/components/Page';
  * @param {Page} page - The '@playwright/test' Page being used by a test
  */
 export class SignIn extends BasePage {
-  readonly title: string = 'Sign In - HPE Machine Learning Development Environment';
+  readonly title: string = `Sign In - ${DevFixture.constants.appTitle}`;
```

Comment:
i'd rather see something like
```
  readonly title = DevFixture.getTitle('Sign In');
```

Reply:
Why? I think I prefer the current way. We will need to leave access to `appTitle` since `blah - ${appTitle}` is not a universal format. (at least User Management is different) so this seems like a util that doesn't always work, and only encapsulates a constant of ` - `.

Reply:
i think if we pass empty string through, then we lop off the preceding `- `. that's probably how webui does it

Reply:
I moved this to base page with a getTitle() function as discussed on slack.